packages/canary/tailwind.config.js

```diff
@@ -74,6 +74,18 @@ module.exports = {
         },
         ai: {
           DEFAULT: 'hsl(var(--ai))'
+        },
+        stage: {
+          DEFAULT: 'hsl(var(--grey-50))'
+        },
+        user: {
+          DEFAULT: 'hsl(var(--grey-40))'
+        },
+        git: {
+          DEFAULT: 'hsl(var(-grey-90))'
+        },
+        log: {
+          DEFAULT: 'hsl(var(--grey-30))'
```

Comment:
these might be better kept in the playground or gitness tailwind config, since they seem to be specific to those apps.

Reply:
Done, moved to playground's tailwind config.